examples/pion-to-pion/docker-compose.yml

```diff
@@ -13,5 +14,6 @@ services:
       - answer
     container_name: offer
     build: ./offer
+    command: offer -answer-address answer:60000
```

Comment:
answer port is 50000 (line 10), isn't it?

Reply:
I am actually not sure if line 10 is needed at all since we only want the two containers to talk to each other.

The flag value `answer:60000` is needed to let the offerer know where to find the answer. By default, this would be `:60000` on localhost (see [`answer/main.go:35`](https://github.com/pion/webrtc/blob/fix/pion-to-pion-docker-compose/examples/pion-to-pion/answer/main.go#L35)). But when both run in different containers, they need the name of the other container instead.

Reply:
I don't see how that is related to the issue this PR is supposed to fix. It may be useful to expose ports if you want to connect from the host machine instead.

Reply:
As you said answer container works on port 60000, so exposing port 50000 does not make sense.